src/foam/u2/ActionView.js

```diff
@@ -313,6 +318,12 @@ foam.CLASS({
         this.buttonState = undefined;
       }
     },
+    {
+      name: 'hasConfirmationModal',
+      factory: function() { 
+        return this.action && this.action.view ? true : false;
```

Comment:
BTW, this would be simpler to just write as:
return this.action && this.action.view

But I don't think this needs to be a property, since developers aren't going to set it directly, are they? Better to make a method or just inline this logic in the one place where it is used.